lib/full-site-editing/edit-site-export.php

```diff
@@ -24,6 +56,9 @@ function gutenberg_edit_site_export() {
 	// Load templates into the zip file.
 	$templates = gutenberg_get_block_templates();
 	foreach ( $templates as $template ) {
+		$updated_content   = _remove_theme_attribute_from_content( $template['content'] );
```

Comment:
No need for declaring `$updated_content` variable here since it's not reused below.

``` 
$template->content = _remove_theme_attribute_from_content( $template['content'] ); 
```

Reply:
👍